cartsquad/apps/accounts/admin.py

```diff
@@ -5,7 +5,7 @@
 from .models import Account
 
 class AccountAdmin(UserAdmin):
-    list_display = ('user_id','email', 'first_name', 'date_joined', 'last_login', 'is_admin', 'is_staff')
+    list_display = ('user_id','email', 'first_name', 'date_joined', 'last_login', 'is_admin', 'is_retailer')
```

Comment:
is_stall is default django field replace this with is_retailer show in django admin